kernel/idle.c

```diff
@@ -151,6 +177,18 @@ void idle(void *unused1, void *unused2, void *unused3)
 	}
 #else
 	for (;;) {
+#if defined(CONFIG_INIT_STACKS)
+		if (k_uptime_get_32() - idle_ts > K_SECONDS(5)) {
```

Comment:
For this kind of periodic logging the period should probably be made configurable through Kconfig. E.g. if the shell is enabled one might want to disable this completely and just rely on the explicit "kernel stacks" command.

Reply:
Agree. Will try to remember to fix once its figured out how to get all the threads enumerated.

Reply:
Yep, must be possible to configure it in build time. Make clear that this is an idle thread and there is not time guarantee.